background_scripts/main.js

```diff
@@ -295,6 +356,28 @@ const BackgroundCommands = {
   moveTabLeft: moveTab,
   moveTabRight: moveTab,
 
+  async setZoom({ tabId, registryEntry }) {
+    const zoomLevel = registryEntry.optionList[0] ?? 1;
+    const newZoom = parseFloat(zoomLevel);
+    // Chrome breaks on values below 0.01 and 10 or above.
```

Comment:
Chrome allows setting the zoom between 1% and 999%, but it will not actually change the visual size of the page beyond 0.25 and 5.00. I decided to allow setting to any value Chrome allows, which seems the most sensible and should help with support for other Chrome based browsers.